modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js

```diff
@@ -125,17 +127,27 @@ export default function EditEntry({
 							`/o/portal-workflow-metrics/v1.0/processes/${appWorkflowDefinitionId}/instances`,
 							{classPKs: [dataRecordId]}
 						).then(({items}) => {
-							setLoading(false);
-
+							let retryCount = 0;
 							if (items.length) {
 								const {id, ...instance} = items.pop();
 
 								const [assignee] = instance.assignees || [];
 
+								if (
```

Comment:
@rafaprax this `retryCount` doesn't represent the count of error tentatives, but count the tentatives of get the updated workflow information, so when `retryCount` will equal to 5  the function execution will finish and the page will be load normally.

Reply:
But we are retrying because the elastic limitation and if we achieve the retryCount == 5 condition it means that, for some reason, the information was not updated yet and we still in the "Bug" condition. I mean, when we achieve the max retries we will continues with a wrong behavior. Makes sense?

Thx

Reply:
So, adding this fallback we will handle this case properly.

Reply:
Yes, makes sense.